SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -3,52 +3,71 @@
 #pragma once
 
 #include "Interop/SpatialStaticComponentView.h"
+#include "Schema/AuthorityIntent.h"
+#include "Schema/ComponentPresence.h"
+#include "Schema/NetOwningClientWorker.h"
+#include "Schema/StandardLibrary.h"
```

Comment:
StandardLibrary.h needed?

Reply:
yeah, includes the definition of the ACL

Reply:
tried forward declaring and removing this. Bunch of other classes start saying EntityACL is ambiguous

Reply:
however just straight up removing it is fine. Not sure where it's being included from then. This feels less explicit, but can remove

Reply:
done

Reply:
Yeah if it is used then better to keep it in